deployment/ccip/changeset/v1_5_1/cs_configure_token_pools.go

```diff
@@ -654,3 +665,41 @@ func ConfigureMultiplePoolLogic(env cldf.Environment, c ConfigureMultipleTokenPo
 	}
 	return finalOutput, nil
 }
+
+// UpdateGroupsOnHybridWithExternalMinterTokenPool updates the groups on hybrid with external minter token pools for a given token across multiple chains.
+func UpdateGroupsOnHybridWithExternalMinterTokenPool(env cldf.Environment, c ConfigureTokenPoolContractsConfig) (cldf.ChangesetOutput, error) {
```

Comment:
This is a different changeset, should have its own config struct and live in a different file